src/Controller/AppController.php

```diff
@@ -244,6 +244,14 @@ function ($item) {
                 return ['name' => $category];
             }, $data['categories']);
         }
+
+        // locked
+        $locked = (string)Hash::get($data, 'locked');
```

Comment:
Do we need this? Is `locked` form field necessary?
(see also comments below)

Reply:
this provides the server side logic, to remove from payload "status" and "uname", when saving a locked object. not necessary, but useful IMHO